tests/test_globwildcards/Snakefile

```diff
@@ -1,11 +1,25 @@
+(IDS,) = glob_wildcards("test.{id}.txt")
```

Comment:
I think the files for this first glob are missing. And further, please add a corresponding test function to tests/tests.py so that the testcase is actually executed.

Reply:
Thanks for your approvel!

test_globwildcards is an existed test, and I just modified a little bit. It is announced in
https://github.com/snakemake/snakemake/blob/bdd732d1f98d0af0362115f74baa728bfc976851/tests/tests.py#L289-L290

The [tests](https://github.com/snakemake/snakemake/tree/main/tests/test_globwildcards/expected-results) are added  11 years ago

Reply:
Ah, sorry. Still too early in the morning for me :-)